docs/content/foundations/token-changes.mdx

```diff
@@ -43,14 +43,14 @@ USWDS is built using a suite of [customizable settings](https://designsystem.dig
 
 ### Shadow Changes
 
-| Shadow Token    | CSS Values (Old to New)                                                                               | Visualization                                                                                                                |
-| :-------------- | :---------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------- |
-| `'shadow-none'` | `'none'`                                                                                              | <div style="box-shadow: none">Shadow None (No Change)</div>                                                                  |
-| `'shadow-1'`    | `'box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.1)'`<br/>`'box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.3)'`     | <div style="box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.1)">Old Shadow 1</div><br/><div className="shadow-1">New Shadow 1</div>  |
-| `'shadow-2'`    | `'box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1)'`<br/>`'box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.3)'`     | <div style="box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1)">Old Shadow 2</div><br/><div className="shadow-2">New Shadow 2</div>  |
-| `'shadow-3'`    | `'box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.1)'`<br/>`'box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.3)'`   | <div style="box-shadow: 0 1px 16px 0 rgba(0, 0, 0, 0.1)">Old Shadow 3</div><br/><div className="shadow-3">New Shadow 3</div> |
-| `'shadow-4'`    | `'box-shadow: 0 12px 24px 0 rgba(0, 0, 0, 0.1)'`<br/>`'box-shadow: 0 12px 24px 0 rgba(0, 0, 0, 0.3)'` | <div style="box-shadow: 0 1px 24px 0 rgba(0, 0, 0, 0.1)">Old Shadow 4</div><br/><div className="shadow-4">New Shadow 4</div> |
-| `'shadow-5'`    | `'box-shadow: 0 16px 32px 0 rgba(0, 0, 0, 0.1)'`<br/>`'box-shadow: 0 16px 32px 0 rgba(0, 0, 0, 0.3)'` | <div style="box-shadow: 0 1px 32px 0 rgba(0, 0, 0, 0.1)">Old Shadow 5</div><br/><div className="shadow-5">New Shadow 5</div> |
+| Shadow Token    | CSS Values (Old to New)                                                                               | Visualization                                                                                          |
+| :-------------- | :---------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------- |
```

Comment:
weren't  you getting rid of the colons?

Reply:
I only changed the one table that broke. This did not seem to break.